scratch_notebooks/test_kepler.jl

```diff
@@ -100,7 +111,7 @@ begin
 	def small_star():
 		t = np.linspace(0, $P, 500)
 	
-		a = $(orbit.aRₛ) * $Rₛ # `exoplanet` stores a in solar units
+		a = $(orbit.aRₛ) #* $Rₛ # `exoplanet` stores a in solar units
```

Comment:
I think this might be a bug in the exoplanet test? They seem to be [passing the semi-major axis](https://github.com/exoplanet-dev/exoplanet/blob/main/tests/orbits/keplerian_test.py#L337-L340) instead of a/R_s. The only reason I caught this is because defining an `orbit` in exoplanet using either the stellar density or m_star and r_star separately seem to give different results, while ours are self-consistent:

**exoplanet:**
```python
from exoplanet.orbits.keplerian import KeplerianOrbit
from batman import _rsky

m_star = 0.151
r_star = 0.189
period = 0.4626413
t0 = 0.2
b = 0.5
ecc = 0.1
omega = 0.1
t = np.linspace(0, period, 500)

orbit = KeplerianOrbit(
    r_star=r_star,
    m_star=m_star,
    period=period,
    t0=t0,
    b=b,
    ecc=ecc,
    omega=omega,
)
orbit.a.eval() # 1.34047805 (in solar radii, based on their G_grav conversion)
orbit.rho_star.eval() # 31.53136891 (g/cc)

orbit2 = KeplerianOrbit(
    rho_star = 31.53136891307936,
    period=period,
    t0=t0,
    b=b,
    ecc=ecc,
    omega=omega,
)
orbit2.a.eval() # 7.09247644 # I think this is actually a/R_star now
orbit2.rho_star.eval() # 31.53136891
```
[G_grav conversion factor](https://github.com/exoplanet-dev/exoplanet/blob/main/src/exoplanet/orbits/constants.py#L18)

Making this small change leads to passing tests now, but just wanted to check with y'all first!

Reply:
I think your explanation is valid, and thankfully it just seems like a unit conversion technicality for the tests. The two orbits ought to be self-consistent. I would actually open an issue over there mentioning the two orbits are discrepant.

Reply:
Thanks for the sanity check! Sure, can do.

Reply:
I've been spotting checking intermediate values calculated with `exoplanet` as I go with the different tests here, but I def can't wait until we have that ALFM 20 boxes ticked off on the TODO!

Reply:
Just heard back! They default to 1 solar radii when only the stellar density is given, my bad! Updating the test now

Reply:
Wow, it looks like they do everything relative to Rsun, including getting positions. I think since we are working in units relative to Rstar, feeding `aRₛ` instead of `a` (in units of solar radii) to `_rsky` is justified. The tests bear this out as well. Below is a quick comparison between the two methods:
![download](https://user-images.githubusercontent.com/25312320/110048450-e0902a00-7d1d-11eb-9e6e-6b4e5f67ceb6.png)


The bottom curve is the difference after applying the scaling factor (1/Rstar) to our implementation, so I think we are good to go now!

Reply:
I think I also prefer doing it this way so that we don't need to [sprinkle `1 / r_star`](https://github.com/exoplanet-dev/exoplanet/blob/d46b9fd79d4a13e8dd229c282bc2f7706552c06f/tests/orbits/keplerian_test.py#L372) in our tests to get the right scaling, since things are already in units of the star